dbus/model-dbus.xml

```diff
@@ -18,5 +18,33 @@
       <arg type="s" name="name" direction="in" />
       <arg type="i" name="result" direction="out" />
     </method>
+    <!-- Register the given model -->
+    <method name="Register">
```

Comment:
Discussion: We didnt get the ACR process for ml-service model yet.
If the interfaces - set-path/get-path/delete are unnecessary, let's remove these.

Reply:
You're right. I removed the outdated methods and related tests. PTAL